Source/UI/NewUI/PaymentForm/ViewModel/PaymentViewModel.swift

```diff
@@ -4,11 +4,12 @@ protocol PaymentViewModel {
     var billingFormData: BillingForm? { get set }
     var paymentFormStyle: PaymentFormStyle? { get set }
     var billingFormStyle: BillingFormStyle? { get set }
-    var environment: Environment { get set }
+    var supportedSchemes: [Card.Scheme] { get set}
+    var environment: Environment { get set}
     var updateEditBillingSummaryView: (() -> Void)? { get set }
     var updateAddBillingDetailsView: (() -> Void)? { get set }
     var updateExpiryDateView: (() -> Void)? { get set }
     var updateCardNumberView: (() -> Void)? { get set }
-    var updateSecurityCodeView: (() -> Void)? { get set }
+    var updateSecurityCodeView: (([Card.Scheme]) -> Void)? { get set }
```

Comment:
Why is there an input of Card.Scheme list for updateSecurityCodeView ? There's also a `supportedSchemes` property that gives the same property. Are these needed together? Why other update callbacks don't provide arguments to the completion handlers? 🤔

Reply:
Updated